imswitch/imcontrol/model/managers/rotators/TelemetrixRotatorManager.py

```diff
@@ -0,0 +1,222 @@
+        self.__initializeMotor()
+
+    def close(self):
+        """
+        Shutdown board. If the motor is currently moving, it will stop it.
+        """
+        self.__logger.info('Motor shutting down.')
+        try:
+            self.board.stepper_stop(self.motorID)
+        except:
+            pass
+        self.board.shutdown()
+
+    def __initializeBoard(self) -> Union[TelemetrixInterface, MockTelemetrixBoard]:
+        """ Initializes communication with the Telemetrix firmware.
+        If no board is found, a mock object is used instead.
+        """
+        board = None
+        try:
+            self.__logger.info('Initializing Telemetrix interface.')
+            board = TelemetrixInterface()
+            self.__logger.info('Telemetrix interface initialized')
+        except Exception:
+            self.__logger.warning('Failed to initialize Telemetrix board. Setting up mock object.')
+            board = MockTelemetrixBoard()
+        return board
+
+    def __initializeMotor(self) -> None:
+        """ Initializes the board motor pin configuration.
+        If the board handle is a mock object, it will call placeholder methods.
+        """
+        self.motorID = self.board.set_pin_mode_stepper(interface=self.interface.value, **self.pinConfig)
+        self.board.stepper_set_max_speed(self.motorID, self.maximumspeed)
+        self.board.stepper_set_current_position(self.motorID, 0)
+
+        # Rotator controller
+        self.set_rot_speed(self.speed)
+        self.board.stepper_set_acceleration(self.motorID, self.acceleration)
+
+    def set_rot_speed(self, speed: int) -> None:
+        """Change the speed of the motor, can be called
+        from the RotatorController
+
+        Args:
+            speed (int): speed value
+        """
+        self.board.stepper_set_speed(self.motorID, speed)
+        self.speed = speed
+
+    def set_zero_pos(self) -> None:
+        """ Set zero position on the rotator and update value
+        in the widget.
+        """
+        self.board.currentPosition = (0, 0)
+        self.board.stepper_set_current_position(self.motorID, 0)
+        self.sigRotatorPositionUpdated.emit()
+
+    def move_rel(self, angle):
+        """ Method implemented from base class.
+        Performs a movement relative to the current position.
+        """
+        steps = angleToSteps(angle, self._stepsPerTurn)
+        self.board.stepper_move(self.motorID, steps)
+        self.board.stepper_run(self.motorID, self.__moveFinishedCallback)
+
+    def move_abs(self, value, inSteps=False):
+        """Method implemented from base class.
+        Moves the stepper to an absolute position, based on the current
+        stored position (in steps).
+
+        Args:
+            angle (int): rotation angle.
+        """
+        # TODO: move this check to controller
+        if inSteps:
+            steps = value
+        else:
+            steps = angleToSteps(value, self._stepsPerTurn)
```

Comment:
Can this check be moved to the controller as stated in the comment? This would provide an easier time dealing with a single measurement unit in the model APIs

Reply:
Unfortunately, to do that, I would need to change RotatorController+RotationScanController. The default for imswitch is angles, but I think it is better to internally work in steps, which is what the Telemetrix and OPT does. Propagating that is perhaps separate PR?

Reply:
Sorry I forgot to reply to this. Then we'll keep it as it is for now, I don't want to overcomplicate things.